docs/metasploit-framework.wiki/Java-Meterpreter-Feature-Parity-Proposal.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Java Meterpreter Feature Parity
+status: not implemented
+---
+
```

Comment:
Let's nuke for now this as it gets rendered:

<img width="1167" alt="image" src="https://user-images.githubusercontent.com/60357436/169550995-0e0c4734-eecd-40c3-a363-35fa03450511.png">